### PR TITLE
chore: handle unknown flags

### LINKED ISSUE #156: Fails on supplying unknown options
### Describe the Bug

It fails miserably when an unknown option is supplied (we're not depending on any sort of argument parser libraries).

### Steps to Reproduce

`create-rust-webpack -i` (random flag supplied)

### Expected Behavior

An appropriate warning message such that no flags are expected. (The respective case should be validated apppropriately.)

### Actual Behavior

Screenshot from 2019-08-25 12-31-10

### Additional Context

N/A

### PR DESCRIPTION
**Pull request by jamesgeorge007**
closes #156

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
\cc @sendilkumarn

File: .bin/create-rust-webpack.js [hunk 1]
```diff
@@ -32,10 +32,17 @@ if (!isPresent("git --version")) {
...
+if (args.some(arg => arg.includes('-'))) {
+  console.log(`Error: unknown option ${args[0]}`);
+  process.exit(1);
+}
+
 let folderName = '.';
 
-if (process.argv.length >= 3) {
-  folderName = process.argv[2];
+if (args.length >= 3) {
```
Comment (sendilkumarn) [inline .bin/create-rust-webpack.js]: Are you sure about this? At a first glance `args.length >= 3` seems wrong. Can you double check?

Comment (jamesgeorge007) [reply .bin/create-rust-webpack.js]: oops!

**Comment by jamesgeorge007** [comment]
\cc @sendilkumarn

File: .bin/create-rust-webpack.js [hunk 2]
```diff
@@ -32,10 +32,17 @@ if (!isPresent("git --version")) {
...
 
-if (process.argv.length >= 3) {
-  folderName = process.argv[2];
+if (args.some(arg => arg.includes('-'))) {
+  console.log(`Error: unknown option ${args[0]}`);
+  process.exit(1);
+}
+
+let folderName = '.';
+console.log(args.length);
```
Comment (sendilkumarn) [inline .bin/create-rust-webpack.js]: why do we need this?

File: .bin/create-rust-webpack.js [hunk 3]
```diff
@@ -32,10 +32,17 @@ if (!isPresent("git --version")) {
...
+if (args.length >= 1) {
```
Comment (sendilkumarn) [inline .bin/create-rust-webpack.js]: aren't we already slicing this?

**Comment by jamesgeorge007** [comment]
:+1:

**Comment by jamesgeorge007** [comment]
@sendilkumarn hope we're good to land this one :thinking:

File: .bin/create-rust-webpack.js [hunk 4]
```diff
@@ -32,13 +32,18 @@ if (!isPresent("git --version")) {
   process.exit(1);
 }
 
+const args = process.argv.slice(2);
```
Comment (sendilkumarn) [inline .bin/create-rust-webpack.js]: Won't this crash, if the arguments are not expected.

Comment (jamesgeorge007) [reply .bin/create-rust-webpack.js]: oops!

**Comment by jamesgeorge007** [comment]
Hope we're good to go :+1:

**Comment by jamesgeorge007** [comment]
\cc @sendilkumarn

**Comment by sendilkumarn** [comment]
Thanks @jamesgeorge007 for the PR.

**Comment by jamesgeorge007** [comment]
@sendilkumarn it seems this is the right time for a brand new release :rocket:

**Pull request merged**